src/text.c

```diff
@@ -0,0 +1,314 @@
+#include "global.h"
+#include "font.h"
+#include "gf_gfx_loader.h"
+#include "text.h"
+#include "unk_0201F79C.h"
+
+static const struct FontInfo *gFonts;
```

Comment:
sFonts

Reply:
and no struct

Reply:
(I understand this is different in diamond but it will be updated there as well to be correct)

Reply:
Losing `struct` can be done separately

Reply:
it *can* be, but it *shouldn't* be imo